test/dom5_test.js

```diff
@@ -530,25 +536,26 @@ suite('dom5', function() {
       '<!DOCTYPE html>',
       '<link rel="import" href="polymer.html">',
       '<dom-module id="my-el">',
-      '<template>',
+      '<div>', // TODO(justinfagnani): this used to be a template, we should test templates
```

Comment:
I added https://github.com/PolymerLabs/dom5/issues/40 because this is a problem if we update to latest dom5 in tools that need access to template contents (bundler for sure, analyzer probs.)